dask_expr/tests/test_collection.py

```diff
@@ -1921,7 +1941,7 @@ def test_quantile(df):
         df.x.quantile(q=[]).compute()
 
     ser = from_pandas(lib.Series(["a", "b", "c"]), npartitions=2)
-    with pytest.raises(TypeError, match="on non-numeric"):
+    with pytest.raises(ValueError, match="object series"):
```

Comment:
Seems like we ought to check the other methods `_raise_if_object_series` is applied to. Which itself sorta seems like maybe it could be a decorator to make it more visible for which methods it's applied to? (no strong opinions on this though)

Reply:
We have tests in dask/dask, that's why I didn't bother with additional tests.

Yeah it needs a review probably that we do it for all functions, but for now we are consistent with dask/dask, so that's good enough for me for now